pandas/core/arrays/base.py

```diff
@@ -1023,6 +1025,42 @@ def _concat_same_type(
     # of objects
     _can_hold_na = True
 
+    def _accumulate(self, name, skipna=True, **kwargs) -> "ExtensionArray":
+        """
+        Return an ExtensionArray performing the accumulation operation.
+        The underlying data type might change
+        # TODO Clarify
+
+        Parameters
+        ----------
+        name : str
+            Name of the function, supported values are:
+            # TODO Add function signatures
+            - cummin
+            - cummax
+            - cumsum
+            - cumprod
+        skipna : bool, default True
+            If True, skip NA values.
+        **kwargs
```

Comment:
May be for numpy compatibility (axis, dtype, out).

Reply:
What are your expectations adding this? Do you purely want to have it (for this ticket) as accessor, which we'll ignore? If not, I'd guess the impact of axis is None, but dtype might be interesting. I.e for cumsum and integer dtypes, we could also provide the target output type, so not defaulting to (U)Int64. But I don't know if this should be part of this issue

Reply:
The idea would be for something like `np.cumsum(pd.array([1, 2]))` to return an IntegerArray. I'm not sure what all is required for that to work.